src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -81,34 +88,57 @@ foam.CLASS({
           'color',
           'year',
           'city',
-          'discipline',
+          'sport',
           'event',
           'country',
           'gender'
         ];
       }
     },
-    'scroller_'
+    [ 'polymer', false ],
+    'scroller_',
+    'container_',
+    'scrollExtent',
+    'scrollHeight'
   ],
 
   methods: [
+    function init() {
+      this.SUPER();
+      if ( this.polymer ) {
```

Comment:
Rather than have this as a parameter, I think I'd rather see a separate sub-class called the PolymerOlympicDemo that makes the necessary registrations and modifications to be polymer.

The olympic demo is a key example that we want to point users to to base their work off of, so we should keep the code as straight forward and declarative as possible.